ui/src/main/webapp/src/app/components/reports/application-details/application-details.component.ts

```diff
@@ -1,100 +1,83 @@
 import {Component, OnInit, ChangeDetectorRef, ElementRef} from "@angular/core";
-import {ProjectTraversalService} from "../../../services/graph/project-traversal.service";
+import {ApplicationDetailsService} from "./application-details.service";
 import {Params, ActivatedRoute} from "@angular/router";
 import {utils} from "../../../utils";
 import {NotificationService} from "../../../services/notification.service";
-import {PersistedProjectModelTraversalModel} from "../../../generated/tsModels/PersistedProjectModelTraversalModel";
-import {PersistedTraversalChildFileModel} from "../../../generated/tsModels/PersistedTraversalChildFileModel";
-import {ClassificationModel} from "../../../generated/tsModels/ClassificationModel";
-import {InlineHintModel} from "../../../generated/tsModels/InlineHintModel";
-import {TechnologyTagModel} from "../../../generated/tsModels/TechnologyTagModel";
-import {ProjectModel} from "../../../generated/tsModels/ProjectModel";
-import {OrganizationModel} from "../../../generated/tsModels/OrganizationModel";
-import {GraphJSONToModelService} from "../../../services/graph/graph-json-to-model.service";
-import {ArchiveModel} from "../../../generated/tsModels/ArchiveModel";
 import {Http} from "@angular/http";
-import {IdentifiedArchiveModel} from "../../../generated/tsModels/IdentifiedArchiveModel";
-import {TaggableModel} from "../../../generated/tsModels/TaggableModel";
-import {compareTraversals, compareTraversalChildFiles, comparePaths} from "../file-path-comparators";
+import {compareTraversals, compareTraversalChildFiles} from "../file-path-comparators";
 import {ApplicationGroup} from "windup-services";
 import {TagFilterService} from "../tag-filter.service";
-import {TagSetModel} from "../../../generated/tsModels/TagSetModel";
-import {forkJoin} from "rxjs/observable/forkJoin";
-import {TypeReferenceStatisticsService} from "../type-reference-statistics.service";
+import {TypeReferenceStatisticsService} from "./type-reference-statistics.service";
 import {TagDataService} from "../tag-data.service";
 import {TreeData} from "../../js-tree-angular-wrapper.component";
+import {ProjectTraversalReducedDTO} from "windup-services";
+import {ApplicationDetailsDTO} from "windup-services";
+import {FileReducedDTO} from "windup-services";
+import {HintReducedDTO} from "windup-services";
+import {chartColorScheme} from "../color-schemes";
```

Comment:
That is what I am hoping. :) I don't particularly care what scheme we decide on, but it would be nice to make it easy for us to be consistent.